Fix build on sparc64:

gtest.cc:5342:7:
 error: 'raise' was not declared in this scope
       raise(SIGTRAP);

https://github.com/google/googletest/pull/4397

Index: third_party/googletest/googletest/src/gtest.cc
--- third_party/googletest/googletest/src/gtest.cc.orig
+++ third_party/googletest/googletest/src/gtest.cc
@@ -43,6 +43,7 @@
 #include <algorithm>
 #include <chrono>  // NOLINT
 #include <cmath>
+#include <csignal>
 #include <cstdint>
 #include <cstdlib>
 #include <cstring>
